### DISCUSSION TITLE
How are the CSS Variables Retrieved

### DISCUSSION THREAD

**Discussion by tajhans** [monkeytypegame/monkeytype#2342] [Questions] [answered]
I was wondering how the theme variables are retrieved. I need this for an project I am working on and since Monketype already does this I thought I might go here to ask. Thanks!

**Comment by Miodec** [comment, marked as answer]
Im assuming you mean getting the CSS variables in JS

https://github.com/Miodec/monkeytype/blob/master/src/js/theme-colors.js

**Reply by tajhans** [reply]
Thanks but what is the document for document.body. Which file does it pick.

**Reply by Miodec** [reply]
Document.body is just a global javascript thing. The styles this file gets will be whatever is loaded (in the html header)
